lib/puppet/provider/consul_prepared_query/default.rb

```diff
@@ -179,7 +181,12 @@ def flush
         },
         "DNS"    => {
           "TTL" => "#{ttl}s"
-        }})
+        },
```

Comment:
Are you sure this should be part of this json should be included no matter what?

What about users of consul before 6.4? What if they don't want to make a template?